docs/src/pages/HomePage/HomePage.js

```diff
@@ -75,7 +75,7 @@ const CARD_CONTENTS = [
   {
     key: 'using-backpack',
     title: 'Using Backpack',
-    href: ROUTES.GETTING_STARTED,
+    href: ROUTES.USING_BACKPACK,
```

Comment:
Pointing this to the root (`/using`) rather than (`/using/getting-started`) because the latter now comes from autogen and we won't know it ahead of time.